zerver/tests/test_signup.py

```diff
@@ -1470,6 +1467,38 @@ def test_signup_already_active(self) -> None:
         result = self.client_get(result.url)
         self.assert_in_response("You've already registered", result)
 
+    def test_signup_with_used_email(self) -> None:
+        """
+        Check if signing up with an email used in another realm succeeds.
+        """
+        email = self.example_email('hamlet')
+        password = "newpassword"
+        realm = get_realm('lear')
+        do_set_realm_property(realm, 'default_language', u"de")
```

Comment:
I take it this is borrowed from `test_user_default_language_and_timezone` above. :-)

We're already testing the `default_language` logic there, so there's no need to repeat that here -- the test will be clearer without it.

A more minimal example for the setup is `test_signup_with_default_stream_group` below.

Reply:
Yeah, we can definitely remove that.